ginga/Control.py

```diff
@@ -850,16 +866,22 @@ def add_image(self, imname, image, chname=None, silent=False):
         if chname is None:
             channel = self.get_channelInfo()
             if channel is None:
-                raise ValueError("Need to provide a channel name to add the image")
+                raise ValueError("Need to provide a channel name to add "
+                                 "the image")
             chname = channel.name
 
+        # Initialize MODIFIED header keyword for image-modified callback.
+        # This is for consistent keyword display in Contents, and used in
+        # 'image-modified' callback.
+        image.metadata['header']['MODIFIED'] = None
```

Comment:
Now I recall why I had to stick MODIFIED in the header. This is so Contents can pick it up for display. Contents plugin assumes all columns info can be extracted from the header.